libsolidity/interface/CompilerStack.h

```diff
@@ -338,6 +338,10 @@ class CompilerStack: public langutil::CharStreamProvider, public evmasm::Abstrac
 	/// Prerequisite: Successful call to parse or compile.
 	Json const& storageLayout(std::string const& _contractName) const;
 
+	/// @returns a JSON representing the transient storage layout of the contract.
+	/// Prerequisite: Successful call to parse or compile.
+	Json const& transientStorageLayout(std::string const& _contractName) const;
```

Comment:
I think you need another rebase, looks like you pulled in changes from the storage layout PR.

Reply:
Fixed.